docs/apis-tools/java-client/index.md

```diff
@@ -6,7 +6,7 @@ description: "Provide a job worker that handles polling for available jobs, use
 ---
 
 :::info
-Starting with 8.7, the Zeebe Java client will become the new Camunda Java client. This transition brings a new Java client structure designed to enhance the user experience and introduce new features while maintaining compatibility with existing codebases.
+Starting with 8.8, the Zeebe Java client will become the new Camunda Java client. This transition brings a new Java client structure designed to enhance the user experience and introduce new features while maintaining compatibility with existing codebases.
```

Comment:
@houssain-barouni, these are the 8.8 docs for the Java client. Shouldn't we completely adjust them to showcase the Camunda Client?

Reply:
Sorry for the spam. I think the PR was not up-to-date yet when I looked at this It's all updated.